core/dslmcode/shared/drupal-7.x/modules/elmsln_contrib/cis_connector/modules/features/cis_section/cis_section.views_default.inc

```diff
@@ -153,14 +161,6 @@ function cis_section_views_default_views() {
   $handler->display->display_options['fields']['edit_node']['field'] = 'edit_node';
```

Comment:
export this one from inside the courses stack / mooc for now. otherwise the field for instructional outline in use gets lost; which is the most improtant part of this to IDs right now